ams_app/templates/ams_app/search.html

```diff
@@ -0,0 +1,154 @@
+{% extends "ams_app/base.html" %}
+{% block title %}Assignment Management System{% endblock %}
+{% block content%}
+{% block assignments %}{% endblock %}
+
+{% if assignments %}
+  <div class="body">
+    <div class="sub-body">
+      <div class="row">
+        <div class="col s2">
+          <a class="btn-floating btn-large waves-effect waves-light red add-assignment modal-trigger" href="#modal1"><i class="material-icons">add</i></a>
+        </div>
+        <form method="GET" action="{% url 'search_assignments' %} ">
+          <div class="input-field col s8 search">
+            <i class="material-icons right">search</i>
+            <input id="icon_prefix" type="text" name="search" placeholder="Search Assignments" class="validate">
+          </div>
+          {% csrf_token %}
```

Comment:
You don't need "csrf_token" for GET requests, else the token will be displayed in your browser